### DISCUSSION TITLE
Genre Management?

### DISCUSSION THREAD

**Discussion by argash** [advplyr/audiobookshelf#294] [Ideas]
Maybe I'm missing something but is there an easy way to manage/cleanup genres? Mine are a bit of a mess after doing a ton of scraping. 
![image](https://user-images.githubusercontent.com/293324/148324080-674d3a33-b61b-49de-abd6-3c8e67669e5b.png)

**Comment by advplyr** [comment]
Not at the moment. The best way is to filter by genre then batch edit.

**Comment by maxcovergold** [comment]
+1 for some sort of Genre management.  Would be great if could clear all genres, as they're a mess of uselessness, then a quick match just for genre fields on all books
